sentry-ruby/lib/sentry/breadcrumb.rb

```diff
@@ -47,7 +47,7 @@ def to_hash
     # @param message [String]
     # @return [void]
     def message=(message)
-      @message = message ? message.byteslice(0..Event::MAX_MESSAGE_SIZE_IN_BYTES) : ""
+      @message = message && Utils::EncodingHelper.valid_utf_8?(message) ? message.byteslice(0..Event::MAX_MESSAGE_SIZE_IN_BYTES) : ""
```

Comment:
Perhaps, we might want to log a debug message if the message is not UTF-8?

Just something to consider, feel free to ignore me here :)